common/src/main/java/com/microsoft/identity/common/internal/commands/BaseCommand.java

```diff
@@ -34,11 +34,13 @@
 import java.util.Collections;
 import java.util.List;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.experimental.SuperBuilder;
 
 // Suppressing rawtype warnings due to the generic types CommandCallback, BaseController
+@SuppressFBWarnings("RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE")
```

Comment:
Hmm.. this should already be excluded (already defined in config/spotbugs/exclude.xml)

Reply:
and URF_UNREAD_FIELD too.